vm/devices/get/guest_emulation_device/src/lib.rs

```diff
@@ -308,8 +325,6 @@ pub struct GedChannel<T: RingMem = GpadlRingMem> {
     vtl0_start_report: Option<Result<(), Vtl0StartError>>,
     #[inspect(with = "Option::is_some")]
     modify: Option<mesh::OneshotSender<Result<(), ModifyVtl2SettingsError>>>,
-    // TODO: allow unused temporarily as a follow up change will use it to
-    // implement AK cert renewal.
     #[inspect(skip)]
     #[allow(dead_code)]
```

Comment:
>#[allow(dead_code)] [](http://example.com/codeflow?start=4&length=19)

should we have removed this in main?

Reply:
ah I missed this. But it's already removed on main https://github.com/microsoft/openvmm/blob/a1271ed3049a53f3dbd07e1b6ce6ca22e48fd799/vm/devices/get/guest_emulation_device/src/lib.rs#L334